ReportingServicesTools/Functions/CatalogItems/Rest/Get-RsRestItemDataModelParameters.ps1

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) 2017 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function Get-RsRestItemDataModelParameters
+{
+    <#
+        .SYNOPSIS
+            This script fetches the Data Model Parameters related to a Catalog Item report from the Report Server
+
+        .DESCRIPTION
+            TThis script fetches the Data Model Parameters related to a Catalog Item report from the Report Server
+
+        .PARAMETER RsItem
+            Specify the location of the catalog item whose data model parameters should be fetched.
+
+        .PARAMETER ReportPortalUri
+            Specify the Report Portal URL to your Power Bi Report Server Instance.
+
+        .PARAMETER RestApiVersion
+            Specify the version of REST Endpoint to use. Valid values are: "v2.0".
+
+        .PARAMETER Credential
+            Specify the credentials to use when connecting to the Report Server.
+
+        .PARAMETER WebSession
+            Specify the session to be used when making calls to REST Endpoint.
+
+        .EXAMPLE
+            Get-RsRestItemDataModelParameters -RsItem "/MyPbixReport1"
+
+            Description
+            -----------
+            Fetches data model parameter information associated to "MyReport" catalog item found in "/" folder from the Report Server located at http://localhost/reports.
+
+        .EXAMPLE
+            Get-RsRestItemDataModelParameters -RsItem "/MyReport" -WebSession $session
+
+            Description
+            -----------
+            Fetches data model parameter information associated to "MyReport" catalog item found in "/" folder from the Report Server located at specificed WebSession object.
+
+        .EXAMPLE
+            Get-RsRestItemDataModelParameters -RsItem "/MyReport" -ReportPortalUri http://myserver/reports
+
+            Description
+            -----------
+            Fetches data model parameter information associated to "MyReport" catalog item found in "/" folder from the Report Server located at http://myserver/reports.
+    #>
+
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory = $True)]
+        [Alias('ItemPath','Path')]
```

Comment:
alias is usually only for backcomp in old cmdlets, for new the recommendation is to just keep the latest